gitbase-playground/templates/deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
               value: "{{ .Values.settings.bblfshdServer.address }}:{{ default "9432" .Values.settings.bblfshdServer.port }}"
             - name: GITBASEPG_ENV
               value: "{{ .Values.gitbasePlayground.env }}"
+            - name: GITBASEPG_FOOTER_HTML
```

Comment:
Please enclose this in a conditional to avoid adding an empty env variable if `gitbasePlayground.footerHTML` is empty